migrate/repo_checks.py

```diff
@@ -75,6 +78,83 @@ def dry_run(self):
         raise NotImplementedError
 
 
+class EnsureLabels(Check):
+    """
+    All repos in the org should have certain labels.
+    """
+
+    def __init__(self, api: GhApi, org: str, repo: str):
+        super().__init__(api, org, repo)
+        # A list of labels mapped to their hex colors
+        # so that they are the same color in all the repos.
+        # Relevant API Docs: https://docs.github.com/en/rest/issues/labels#create-a-label
+        self.labels = {
+            ":hammer_and_wrench: maintenance": "169509",
+        }
+
+    def is_relevant(self):
+        return not is_security_private_fork(self.api, self.org_name, self.repo_name)
+
+    def check(self):
+        """
+        See if our labels exist.
+        """
+        labels = chain.from_iterable(
+            paged(
+                self.api.issues.list_labels_for_repo,
+                self.org_name,
+                self.repo_name,
+                per_page=100,
+            )
+        )
+
+        existing_labels = {label.name: label.color for label in labels}
+
+        self.missing_labels = []
+        self.incorrectly_colored_labels = []
+        for label, color in self.labels.items():
```

Comment:
Another fix could be to normalize existing labels with similar names

For example, if `Good-First-Issue` is a label in the repo, but we want the repo to have `🏆 good first issue`, then rename the existing label using `update_label` instead of creating a dupe label. The benefit is that we would avoid having a bunch of issues with similarly-named labels that needed to be migrated to the canonical label name.

The comparison logic could be something like: if you lowercase the labels and remove special characters, are the labels equal? If so, rename instead of creating.